app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    _, old_name, new_name = command.split()
```

Comment:
Code Efficiency: Ensure that the `mv` input command is correct by validating the length of the split result and handling cases where the command format is incorrect.